mealvoucher/src/main/res/values/strings.xml

```diff
@@ -11,4 +11,8 @@
     <string name="checkout_meal_voucher_card_number_hint" translatable="false">Card number</string>
     <string name="checkout_meal_voucher_expiry_date_hint" translatable="false">Expiry date</string>
     <string name="checkout_meal_voucher_security_code_hint" translatable="false">Security code</string>
+
+    <string name="checkout_meal_voucher_expiry_date_not_valid" translatable="false">Invalid expiry date</string>
```

Comment:
Shouldn't this be part of the strings.xml.tt file? So it doesn't get removed when we update all the strings

Reply:
We plan to update all strings at once in another PR, that's why for now we have these here.